docs/source/transaction_family_specifications/integerkey_transaction_family.rst

```diff
@@ -0,0 +1,134 @@
+***************************************************
+IntegerKey Transaction Family Specification
+***************************************************
+
+Overview
+=========
+The IntegerKey transaction family allows users to set, increment, and decrement
+the value of entries stored in state. 
+
+An IntegerKey family transaction request is defined by the following values:
+
+* A verb which describes what action the transaction takes 
+* A name of the entry that is to be set or changed
+* A value by which the entry will be set or changed
+
+The 'set' verb is used to create new entries. The initial value of the entry will be 
+set to the value specified in the transaction request. The 'inc' and 'dec' verbs are 
+used to change the value of existing entries in state.
+
+State
+=====
+This section describes in detail how IntegerKey transaction information is stored 
+and addressed.
+
+The address values of IntegerKey transaction family entries are stored in state as
+a cbor encoded dictionary, with the key being the *Name* and the value being an integer 
+*Value*.
+
+.. code-block:: python
+
+    cbor({    
```

Comment:
^ wasn't a good comment, but was more information. I would reference the cbor specification at https://tools.ietf.org/html/rfc7049 and say that we are using a definite map, with that snippet of code up above, and maybe give examples of possible bytes stored in state. So using a python3 repl, b'\xa1cfoo\x13' = cbor.dumps({'foo': 13})